version.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
lb.yaml format is:

     <repo name>:
       <component name>:
         - <dependency>